ee/ucp/admin/install/cloudproviders/install-on-azure.md

```diff
@@ -56,6 +56,19 @@ You must meet the following infrastructure prerequisites to successfully deploy
   needed as part of the UCP prerequisites. If you are using a separate Resource
   Group for the networking components, the same Service Principal will need
   `Network Contributor` access to this Resource Group.
+- Kubernetes pods integrate into the underlying Azure networking stack, from
+  an IPAM and routing perspective with the Azure CNI IPAM module. Therefore
+  Azure Network Security Groups (NSG) impact pod to pod communication. End users
```

Comment:
Suggest changing to: "Users may expose containerized services on a range of underlying ports, resulting in a manual process to open an NSG port each time a new containerized service is deployed on to the platform."